specification/components/schemas/headers/response/ContentTypeFileUpload.yaml

```diff
@@ -5,7 +5,6 @@ schema:
   type: string
   enum:
   - 'text/plain'
-  - 'text/html'
```

Comment:
We don't support the existing HTML files that were uploaded before this fix?

Reply:
Good spot. The OAS update ticket had a typo in it, but story states HTML downloads prior to ERSSUP-60430 will still be supported. Will update shortly.